### PR TITLE
Error boundary Admin

### PR DESCRIPTION
**Pull request by vasylshpak**
Add error boundary for Admin panel
- routes
- boundary
- redux

### CODE REVIEW

**Review by nadiyafomenko — APPROVED** [review]

File: src/redux/error/error.reducer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { SET_ERROR } from './error.types';
+
+const initialState = {
+  error: null
+};
+
+const errorReducer = (action, state = initialState) => {
```
Comment (VitalikVoicix) [inline src/redux/error/error.reducer.js:7]: it seems that there must be another order(state,=>action)

Comment (vasylshpak) [reply src/redux/error/error.reducer.js:7]: Thanks for your attention, that's true but then we got this code smell.
image

**Review by VitalikVoicix — APPROVED** [review]

**Review by dieie32 — APPROVED** [review]

File: src/redux/error/error.reducer.js (hunk 1 above)
Comment (nadiyafomenko) [reply src/redux/error/error.reducer.js:7]: you can assign empty object to avoid this warning

Comment (vasylshpak) [reply src/redux/error/error.reducer.js:7]: thank you, didn't know about that

Comment (vasylshpak) [reply src/redux/error/error.reducer.js:7]: @VitalikVoicix @nadiyafomenko  fixed it, thank you

**Pull request merged**